index.src.html

```diff
@@ -2515,6 +2558,7 @@
 dictionary VideoFrameInit {
   unsigned long long duration;  // microseconds
   long long timestamp;          // microseconds
+  AlphaOption alpha = "keep";
 };
 
 dictionary VideoFramePlaneInit {
```

Comment:
If the outcome of pending discussion is that we keep hasAlpha, we should probably add hasAlpha to this structure as well as a signal for how to interpret the data (e.g. you now have an extra plane for planar formats)